added desktop logo styles to src/components/TheHeader.vue

AlexVCS opened the pull request:
<!-- Thanks for contributing to Cuttle! 🎉 -->

## Issue number
#578 

Relevant [issue number](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- Resolves #578 

## Please check the following

- [ Think so] Do the tests still pass? (see [Run the Tests](https://github.com/cuttle-cards/cuttle/blob/main/docs/setup-development.md#run-the-tests))
- [✅ ] Is the code formatted properly? (see [Linting (Formatting)](https://github.com/cuttle-cards/cuttle/blob/main/docs/setup-development.md#linting-formatting))
- For New Features:
  - [No ] Have tests been added to cover any new features or fixes?
  - [ No] Has the documentation been updated accordingly?

## Please describe additional details for testing this change

itsalaidbacklife reviewed (commented):
Thanks for your contribution! I have one nitpick regarding the style block syntax, and then can you please run `npm run version:patch` one time and push again? This will update the package version a patch number.

Nice work! Once the above is done this can be merged and deployed to production (you can see what the current version is live in prod at https://cuttle.cards/health). If you'd like to make another contribution, please feel free to grab any issue that interests you. If you'd like help finding one, or to discuss one you're working on/considering, you can chat with me and the other devs in the #code channel of [our discord](https://discord.gg/6vQCFa6nvP).

AlexVCS commented:
> Thanks for your contribution! I have one nitpick regarding the style block syntax, and then can you please run `npm run version:patch` one time and push again? This will update the package version a patch number.
> 
> Nice work! Once the above is done this can be merged and deployed to production (you can see what the current version is live in prod at https://cuttle.cards/health). If you'd like to make another contribution, please feel free to grab any issue that interests you. If you'd like help finding one, or to discuss one you're working on/considering, you can chat with me and the other devs in the #code channel of [our discord](https://discord.gg/6vQCFa6nvP).

Thanks for the feedback! Think I have fixed the version now, I initially ran into an error when I tried to do it but was able to when I ran git stash. The typo is fixed and I added the line at the end.

itsalaidbacklife reviewed (approved):
Nice work! Thanks for your contribution
